Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax = docker/dockerfile:1.4
 
 # Make sure RUBY_VERSION matches the Ruby version in .ruby-version and Gemfile
-ARG RUBY_VERSION=3.2.0
+ARG RUBY_VERSION=3.2.2
```

Comment:
Spent 10 minutes trying to golf this to use a `cat .ruby-version` but no luck.

Reply:
🏌️‍♂️